src/main/java/az/kontakthome/mskontakthome/dao/entity/ModelEntity.java

```diff
@@ -0,0 +1,26 @@
+package az.kontakthome.mskontakthome.dao.entity;
+
+import lombok.*;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "models")
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class ModelEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String name;
+    private Boolean isArchive = false;
```

Comment:
is archiveleri sil 3 entitydede bizde day delete methodu olmayacaq deye bunada ehtiyac qalmir